docs/data_access.rst

```diff
@@ -106,32 +108,19 @@ resulting outputs pass all of the data validation tests we've defined, the outpu
 automatically uploaded to the `AWS Open Data Registry
 <https://registry.opendata.aws/catalyst-cooperative-pudl/>`__, and used to deploy a new
 version of Datasette (see above). These nightly build outputs can be accessed using the
-AWS CLI, or programmatically via the S3 API. They can also be downloaded directly over
-HTTPS using the following links:
+AWS CLI, or programmatically via the S3 API. If you don't want to mess with the API
+or CLI, you can also download the data directly over HTTPS. The download links for
+each table's Parquet file can be found in
+the :doc:`PUDL data dictionary page </data_dictionaries/pudl_db>`.
+
+These are the download links for the PUDL and raw FERC SQLite databases:
 
 Fully Processed SQLite Databases
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * `Main PUDL Database <https://s3.us-west-2.amazonaws.com/pudl.catalyst.coop/nightly/pudl.sqlite.zip>`__
 * `US Census DP1 Database (2010) <https://s3.us-west-2.amazonaws.com/pudl.catalyst.coop/nightly/censusdp1tract.sqlite.zip>`__
 
-Hourly Tables as Parquet
```

Comment:
I removed this section because now all tables are available as Parquet.

Reply:
This is true, but these tables _aren't_ in SQLite and so I think shouting them out here is still helpful - if people are looking for them they won't be able to find them in the full DB.